packages/location-management/tsconfig.json

```diff
@@ -5,6 +5,6 @@
     "declaration": true,
     "declarationDir": "dist/types"
   },
-  "include": ["**/src"],
+  "include": ["**/src", "../fhir-location-management/src/components/AllLocationListFlat"],
```

Comment:
This should be exported from fhir-location-managent not here.

Reply:
noted